apps/api/v2/src/modules/slots/controllers/slots.controller.ts

```diff
@@ -52,7 +51,6 @@ export class SlotsController {
 
   @Get("/available")
   @ApiOperation({ summary: "Get available slots" })
-  @Throttle({ default: { limit: 300, ttl: seconds(60) } }) // allow 300 requests per minute
```

Comment:
I think we can remove @throttle from schedules controller as well

Reply:
Done in [refactor: remove route specific](https://github.com/calcom/cal.com/pull/16882/commits/f93503ea10671c1ea35d7f381b24cd448a6dddac) @throttles